### PR TITLE
Add mailjet samples

### PR DESCRIPTION
**Pull request by shun-fan**
Remove unused annotation in mailgun

### CODE REVIEW

**Comment by shun-fan** [comment]
@tswast

**Comment by shun-fan** [comment]
seems like spark is failing again

File: appengine/mailjet/README.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Mailjet sample for Google App Engine
```
Comment (lesv) [inline appengine/mailjet/README.md]: This really should say more.

File: appengine/mailjet/src/main/java/com/example/appengine/mailjet/MailjetServlet.java [hunk 2]
```diff
@@ -0,0 +1,52 @@
+package com.example.appengine.mailjet;
```
Comment (lesv) [inline appengine/mailjet/src/main/java/com/example/appengine/mailjet/MailjetServlet.java]: License & copyright?

File: appengine/mailjet/src/main/webapp/WEB-INF/appengine-web.xml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (lesv) [inline appengine/mailjet/src/main/webapp/WEB-INF/appengine-web.xml]: License & Copyright?

File: appengine/mailjet/src/main/webapp/WEB-INF/web.xml [hunk 4]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (lesv) [inline appengine/mailjet/src/main/webapp/WEB-INF/web.xml]: License & Copyright?

File: managed_vms/mailjet/src/main/appengine/app.yaml [hunk 5]
```diff
@@ -0,0 +1,13 @@
+runtime: java
```
Comment (lesv) [inline managed_vms/mailjet/src/main/appengine/app.yaml]: License & Copyright?

File: managed_vms/mailjet/src/main/java/com/example/managedvms/mailjet/MailjetServlet.java [hunk 6]
```diff
@@ -0,0 +1,54 @@
+package com.example.managedvms.mailjet;
```
Comment (lesv) [inline managed_vms/mailjet/src/main/java/com/example/managedvms/mailjet/MailjetServlet.java]: License & Copyright?

**Comment by lesv** [comment]
PTAL

**Comment by lesv** [comment]
Mailjet is failing in travis as well.

**Comment by shun-fan** [comment]
@lesv PTAL

**Comment by lesv** [comment]
LGTM

**Pull request merged**